testData/solutions/win32.json5

```diff
@@ -685,6 +770,63 @@
                         "ZoomIncrement": "REG_DWORD"
                     }
                 },
+                "supportedSettings": {
```

Comment:
It seems that the general schema for changing settings related to the Windows Magnifier includes a restart of the application (in order for the new settings to be applied). This is fine for settings like "Magnification" and "Magnifier position". However, if the user changes the "Magnifier Enabled" value to be "Off", then Windows Magnifier should simply close and should not restart in this case. This will require a special handling for this particular setting. If this turns out to be too difficult, a workaround can be to simply remove the "Magnifier Enabled" setting from the payload sent to PCP (assuming that if the user wants to disable magnification, he can do that by setting the "Magnification" to 1).

Another thing which should be considered when the "Magnifier Enabled" setting is "Off", is what should happen if the user wants to change other magnification related settings (e.g. "Magnifier position"). In that case Windows Magnifier should probably not start or if it does, GPII should instruct the PSP to change the "Magnifier Enabled" setting to become "On".

There may be other applications for which similar situations may arise but I am not familiar with all of them so please double check.

Reply:
That's a very good point, and something that has been overdue... I've created a JIRA here: https://issues.gpii.net/browse/GPII-2685 which will be a the longer term solution and which I'll get to really soon - until then (in the short term) I think we can get by with simply not having the Magnifier Eabled term in the preferences set so they wont show up in the PSP. If @amb26 gives his blessing on this, that is.